frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/parsers/code_block_node_parser.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:appflowy_editor/appflowy_editor.dart';
+
+class CodeBlockNodeParser extends NodeParser {
+  const CodeBlockNodeParser();
+
+  @override
+  String get id => 'code_block';
+
+  @override
+  String transform(Node node) {
+    return '```$node.attributes[id]```';
```

Comment:
I think adding `\n` before and after the line should be better.

![Uploading Screenshot 2023-03-21 at 15.29.16.png…]()

Reply:
ok 👍🏻